core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -994,6 +994,12 @@ public enum Property {
           + " The resources that are used by default can be seen in"
           + " `accumulo/server/monitor/src/main/resources/templates/default.ftl`.",
       "2.0.0"),
+  MONITOR_OBFUSCATE_EXTENTS("monitor.extents.obfuscate", "true", PropertyType.BOOLEAN,
+      "Obfuscates the table extent information displayed in the Monitor if true. Setting"
+          + " this to false will expose data in the row of the keys where tablets split. It"
+          + " is not recommended to set this to false if the Monitor is exposed to entities"
+          + " that should not see this information.",
```

Comment:
Could mention this can only be set in props file.

Reply:
Updated in afb06ae